vsutil.py

```diff
@@ -1,58 +1,63 @@
+VSUtil. A collection of general-purpose VapourSynth functions to be reused in modules and scripts.
 """
+__all__ = ['core', 'fallback', 'frame2clip', 'get_depth', 'get_plane_size', 'get_subsampling', 'get_w', 'get_y',
+           'insert_clip', 'is_image', 'iterate', 'join', 'plane', 'split', 'vs']
+
 from functools import reduce
-from typing import Callable, TypeVar, Union, List, Tuple, Optional
+from mimetypes import types_map
+from os import path
+from typing import Callable, List, Optional, Tuple, TypeVar, Union
+
 import vapoursynth as vs
-import mimetypes
-import os
 
 core = vs.core
 T = TypeVar("T")
 
 
 def get_subsampling(clip: vs.VideoNode) -> str:
     """
-    Returns the subsampling of a clip in human-readable format.
+    Returns the subsampling of a VideoNode in human-readable format.
     """
+    if clip.format.color_family not in (vs.YUV, vs.YCOCG):
+        return ''
     if clip.format.subsampling_w == 1 and clip.format.subsampling_h == 1:
-        css = '420'
+        return '420'
     elif clip.format.subsampling_w == 1 and clip.format.subsampling_h == 0:
-        css = '422'
+        return '422'
     elif clip.format.subsampling_w == 0 and clip.format.subsampling_h == 0:
-        css = '444'
+        return '444'
     elif clip.format.subsampling_w == 2 and clip.format.subsampling_h == 2:
-        css = '410'
+        return '410'
     elif clip.format.subsampling_w == 2 and clip.format.subsampling_h == 0:
-        css = '411'
+        return '411'
     elif clip.format.subsampling_w == 0 and clip.format.subsampling_h == 1:
-        css = '440'
+        return '440'
     else:
-        raise ValueError('Unknown subsampling')
-    return css
+        raise ValueError('Unknown subsampling.')
 
 
 def get_depth(clip: vs.VideoNode) -> int:
     """
-    Returns the bitdepth of a clip as an integer.
+    Returns the bit depth of a VideoNode as an integer.
     """
     return clip.format.bits_per_sample
 
 
 def get_plane_size(frame: Union[vs.VideoFrame, vs.VideoNode], planeno: int) -> Tuple[int, int]:
     """
-    Calculates the size of the plane
-    
-    :param frame:    The frame
-    :param planeno:  The plane
+    Calculates the dimensions (w, h) of the desired plane.
+
+    :param frame:    Can be a clip or frame.
+    :param planeno:  The desired plane's index.
     :return: (width, height)
     """
     # Add additional checks on Video Nodes as their size and format can be variable.
     if isinstance(frame, vs.VideoNode):
         if frame.width == 0:
-            raise ValueError("Cannot calculate plane size of variable size clip. Pass a frame instead.")
+            raise ValueError('Cannot calculate plane size of variable size clip. Pass a frame instead.')
         if frame.format is None:
-            raise ValueError("Cannot calculate plane size of variable format clip. Pass a frame instead.")
+            raise ValueError('Cannot calculate plane size of variable format clip. Pass a frame instead.')
```

Comment:
Why did you change quotation characters here? Do we have any guidelines on this? I see that both styles are still being used. 

(I know my guidelines, where I use single quotes for symbol-like identifiers and double for what could be described as text, but I am just a third party here.)

Reply:
It was either we switched completely to `"` or `'` and I usually go with `'`, it's not a standard defined anywhere but either/or works, just didn't like seeing both.

Reply:
Now that you mention it actually, I missed the strings in `is_image()`, will update soon.

Reply:
Well, it's documented with a single quote so I changed the TypeVar too <https://docs.python.org/3/library/typing.html#generics>